app/Listeners/TaskStatusHistory.php

```diff
@@ -37,8 +37,8 @@ public function handle(\App\Events\TaskStatusHistory $event)
                 ];
             }
 
-            //update task_history if task is paused or resumed without submitted for QA
-            if (key_exists('paused', $newValues) && (!key_exists('submitted_for_qa', $newValues))) {
+            //update task_history if task is paused or resumed without QA in progress
+            if (key_exists('paused', $newValues) && (!key_exists('qa_progress', $newValues))) {
```

Comment:
`qa_progress` is vague - please rename to `qa_in_progress`, sorry I was not explicit the first time